### PR TITLE
fix(gui): update about dialog and menu items in help

### LINKED ISSUE #519: GUI: Updating about dialog
About dialog needs to be updated:

1- Adding Pactus logo (Logs can be found [here](https://github.com/pactus-project/pactus.org/tree/main/website/assets/images/logos))
2- Adding Version from [version](https://github.com/pactus-project/pactus/blob/main/version/version.go) package
3- Website link should be changed to: https://pactus.org/
4- Changing "See the The MIT License (MIT) for details." to "See the MIT License for details."

Please ensure that the icon is embedded inside the code, so we don't need to change the way we release the software.

### PR DESCRIPTION
**Pull request by Ja7ad**
## Description

This PR covers these changes:

- Adding version, logo and comment in about dialog
- Adding menu items for website, explorer and home
- Adding `getMenuItem` function
- Adding and testing cross platform open URL in browser

Fixed #519

### CODE REVIEW

File: cmd/gtk/assets/ui/main_window.ui [hunk 1]
```diff
@@ -103,6 +103,33 @@
                   <object class="GtkMenu">
                     <property name="visible">True</property>
                     <property name="can-focus">False</property>
+                    <child>
+                      <object class="GtkMenuItem" id="id_website_menu">
+                        <property name="visible">True</property>
+                        <property name="can-focus">False</property>
+                        <property name="label" translatable="yes">Website</property>
```
Comment (b00f) [inline cmd/gtk/assets/ui/main_window.ui:110]: You can update the code as follows:
```
<property name="label" translatable="yes">_Website</property>
```
Adding an underscore (_) before "Website" sets "W" as the accelerator key for the menu item. (Alt+W)

File: cmd/gtk/assets/ui/main_window.ui [hunk 2]
```diff
@@ -103,6 +103,33 @@
...
+                      </object>
+                    </child>
+                    <child>
+                      <object class="GtkMenuItem" id="id_explorer_menu">
+                        <property name="visible">True</property>
+                        <property name="can-focus">False</property>
+                        <property name="label" translatable="yes">Explorer</property>
```
Comment (b00f) [inline cmd/gtk/assets/ui/main_window.ui:117]: Same here: `_Explorer`

File: cmd/gtk/assets/ui/main_window.ui [hunk 3]
```diff
@@ -103,6 +103,33 @@
...
+                      </object>
+                    </child>
+                    <child>
+                      <object class="GtkMenuItem" id="id_learn_menu">
+                        <property name="visible">True</property>
+                        <property name="can-focus">False</property>
+                        <property name="label" translatable="yes">Learn</property>
```
Comment (b00f) [inline cmd/gtk/assets/ui/main_window.ui:124]: Same here: `_Learn`

File: cmd/gtk/assets/ui/main_window.ui [hunk 4]
```diff
@@ -116,7 +143,7 @@
                       <object class="GtkMenuItem">
                         <property name="visible">True</property>
                         <property name="can-focus">False</property>
-                        <property name="label">About  _GTK</property>
+                        <property name="label" translatable="yes">About  _GTK</property>
```
Comment (b00f) [inline cmd/gtk/assets/ui/main_window.ui:146]: I can see an extra space here: `About _GTK`

**Review by b00f — APPROVED** [review]

**Comment by Ja7ad** [comment]
close #531

**Review by b00f — APPROVED** [review]

**Pull request merged**